src/System.Net.Http/src/System/Net/Http/HttpClientHandler.Windows.cs

```diff
@@ -442,9 +442,21 @@ public SslProtocols SslProtocols
             _winHttpHandler.Properties :
             _socketsHttpHandler.Properties;
 
-        protected internal override Task<HttpResponseMessage> SendAsync(HttpRequestMessage request,
+        protected internal override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request,
             CancellationToken cancellationToken)
         {
+            if (request == null)
+            {
+                throw new ArgumentNullException(nameof(request), SR.net_http_handler_norequest);
+            }
+
+            var error = await ValidateAndNormalizeRequestAsync(request).ConfigureAwait(false);
```

Comment:
`var` should not be used her when its type is not obvious.

```suggestion
            Exception error = await ValidateAndNormalizeRequestAsync(request).ConfigureAwait(false);
```